### DISCUSSION TITLE
MCRALS plot of St *

### DISCUSSION THREAD

**Discussion by kiyoAsakura** [spectrochempy/spectrochempy#730] [Q&A] [answered]
My spectra contain positive and negative data.  But when I applied MCRALS analysis and plotted the St spectra by mcr.St.plot,  I only have the positive part.  Should I give some parameters such as ylim()?

Thanks.

**Comment by atravert** [comment, marked as answer]
Hello,

The plot() method displays the totality of spectra, including the negative part. Have you checked whether mcr.St actually contained negative values ?

By default, MCRALS considers that spectra are non-negative: have you set [`nonnegSpec`](https://www.spectrochempy.fr/latest/reference/generated/spectrochempy.MCRALS.html#spectrochempy.MCRALS.nonnegSpec) to `[]` (i.e. all spectra can be negative) or, for example, `[1]` to indicate that the second spectrum (index `1`) can be negative while the others (`0`, `2`,...) are non-negatives ?

Best regards
Arnaud

**Comment by kiyoAsakura** [comment]
Dear Arnaud
Thank you very much for your reply.  
I am now analyzing EXAFS chi(k) data which contain positive and negative.  I will try it by adding a positive constant value. 
Thank you again for your help.
Best regards
Kiyotaka
